src/components/implementation/no_interface/cFE_booter/osapi.c

```diff
@@ -0,0 +1,457 @@
+#include <cos_component.h>
+#include <cos_defkernel_api.h>
+
+#include <sl.h>
+#include <sl_consts.h>
+
+#include "gen/osapi.h"
+#include "gen/cfe_psp.h"
+#include "gen/cfe_time.h"
+#include "gen/common_types.h"
+
+#include "cFE_util.h"
+
+/*
+** Initialization of API
+*/
+int have_initialized = 0;
+
+int32
+OS_API_Init(void)
```

Comment:
It's that way because the OSAL unit tests expect this to be idempotent. We can't duplicate initialization, so I'm not sure what a better solution would look like.